scripts/globals/mobskills/super_buff.lua

```diff
@@ -0,0 +1,19 @@
+---------------------------------------------
+-- Spike Flail
+-- Deals extreme damage in a threefold attack to targets behind the user.
+---------------------------------------------
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/settings")
```

Comment:
I'm not sure why this was in the original Super Buff, but settings aren't used in this one!